RobloxIPFinder/Form1.cs

```diff
@@ -19,34 +21,54 @@ public Form1()
         }
 
         // Made by daymon - twitter.com/disband
+
+        // Fixed by Nick.
+        
         private void simpleButton1_Click(object sender, EventArgs e)

```

Comment:
Doing this in the UI thread is such a 16-year-old me move, that's on me :')

Reply:
Lol. As 15 I only adjusted to what you did.